src/app/projects/edit-project/edit-project.component.html

```diff
@@ -0,0 +1,29 @@
+<form [formGroup]="projectForm" class="edit-project-form">
+  <div *ngIf="errorMessage" class="edit-project-error">{{errorMessage}}</div>
+  <div fxLayout="column">
+    <div fxFlex>
+      <mat-form-field class="edit-project-form-field">
+        <input matInput placeholder="Name" formControlName="name" [required]="true">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="edit-project-form-field">
+        <input matInput placeholder="Customer" formControlName="customer">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="edit-project-form-field">
+        <input matInput placeholder="Industry sector" formControlName="industrySector">
+      </mat-form-field>
+    </div>
+    <div fxFlex>
+      <mat-form-field class="edit-project-form-field">
+        <input matInput placeholder="Description" formControlName="description">
+      </mat-form-field>
+    </div>
+  </div>
+  <div class="edit-project-actions">
+    <button mat-raised-button color="primary" (click)="editProject()" [disabled]="!projectForm.valid" class="edit-project-button">Edit project</button>
```

Comment:
I think we could use `projectForm.pristine` to prevent user from updating of a project without any changes in the form.
@svetivanova what do you think about it?

Reply:
done